xbmc/addons/AddonCallbacksAddon.cpp

```diff
@@ -543,6 +543,7 @@ static void CFileItemListToVFSDirEntries(VFSDirEntry* entries,
     entries[i].label = strdup(items[i]->GetLabel().c_str());
     entries[i].path = strdup(items[i]->GetPath().c_str());
     entries[i].size = items[i]->m_dwSize;
+    //entries[i].mtime = items[i]->m_dateTime;
```

Comment:
Rebase error? The time field was removed from [kodi_vfs_types.h](https://github.com/xbmc/xbmc/blob/master/xbmc/addons/include/kodi_vfs_types.h)